DEVELOPER_GUIDE.md

```diff
@@ -4,6 +4,9 @@
     - [Install Prerequisites](#install-prerequisites)
       - [Node.js](#nodejs)
       - [Docker](#docker)
+      - [Yarn](#yarn)
+    - [Installing dependencies](#installing-dependencies)
```

Comment:
TOC also needs fixing.

Reply:
Oh, sorry. Should be fixed with the recent commit